githubclient/__init__.py

```diff
@@ -42,12 +46,13 @@ def _get(self, uri, paginated=False, page_size=100, max_pages=100, querystring=N
                 results = page_of_results
             if 'Link' not in response.headers:
                 break
-            links = GitHubClient.decompose_link_header(response.headers.get('Link'))
+            links = GitHubClient.decompose_link_header(
+                response.headers.get('Link'))
             if 'next' not in links:
                 break
             uri = links.get('next')
         if pages == max_pages:
-            print("WARNING: max_pages of {} exceeded.".format(max_pages))
+            print "WARNING: max_pages of {} exceeded.".format(max_pages)
```

Comment:
We probably want to leave the parentheses and add `from __future__ import print_function` if we want to maintain Python 2.7 compatibility.

Reply:
Oh, sorry, I thought 2.7 python was OK as Fedora uses 2.7 by default with pylint?

I will add them back in a little if that is OK?  Thanks for letting me know

Reply:
```
python --version
Python 2.7.14

python
Python 2.7.14 (default, Mar 14 2018, 13:36:31) 
[GCC 7.3.1 20180303 (Red Hat 7.3.1-5)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> print "Hello without parens"
Hello without parens
>>> print ("One more hello")
One more hello
>>> quit()

``

Reply:
In that example, the second print is not actually the print_function from Python 3, it is the old version but just printing a tuple because it has a space.